packages/babel-check-binary-expressions/src/index.js

```diff
@@ -1,6 +1,24 @@
-export default function(left, right) {
-  // eslint-disable-next-line no-undef
-  if (left instanceof BigInt && right instanceof BigInt) {
+import isNumber from "lodash/isNumber";
+import isString from "lodash/isString";
+
+function isBigInt(x) {
+  return x instanceof global.BigInt;
+}
+
+export default function(left, right, operator) {
```

Comment:
I'm not sure why you take the operator as an argument, but then you just use `+`.